{{ cookiecutter.formal_name }}/app/src/main/java/org/beeware/android/MainActivity.java

```diff
@@ -20,6 +20,8 @@
 import org.beeware.rubicon.Python;
 import org.jetbrains.annotations.Nullable;
 
+import {{ cookiecutter.bundle }}.{{ cookiecutter.app_name }}.R;
```

Comment:
I'm fairly certain this will need to be `module_name`, not `app_name` - and there will probably be a need for a change to `bundle`, too. An app with a full bundle ID of `com.keith-magee.hello-world` will reveal the problem; this will need to be an import of `com.keith_magee.hello_world.R`.

Reply:
This may require an update to briefcase to inject a "dashes to underscores" version of the bundle.

Reply:
👍